src/Particles/babylon.particleSystem.ts

```diff
@@ -484,15 +497,23 @@
             this._started = true;
             this._stopped = false;
             this._actualFrame = 0;
+            if (this.subEmitters && this.subEmitters.length != 0) {
+                this.activeSubSystems = new StringDictionary<ParticleSystem>();
+                this.stockSubSystems = new StringDictionary<Array<ParticleSystem>>();
+            }
         }
 
         /**
          * Stops the particle system.
          */
-        public stop(): void {
+        public stop(stopSubEmitters = true): void {

```

Comment:
parameter comment (hint: run gulp typedoc-check to see if you have issues with doc)